packages/authenticator/amplify_authenticator/example/ios/Runner/Info.plist

```diff
@@ -47,5 +47,9 @@
 	<true/>
 	<key>UIApplicationSupportsIndirectInputEvents</key>
 	<true/>
+  <key>LSApplicationQueriesSchemes</key>
+  <array>
+    <string>otpauth</string>
+  </array>
```

Comment:
This should be removed. The Authenticator app handles the URL not the user's app.